lib/src/pages/code_verification.dart

```diff
@@ -174,6 +174,9 @@ class _CodeVerificationState extends State<CodeVerification> {
         widget.prefs.setBool('loggedIn', true);
         // save token
         widget.prefs.setString('jwt_token', rsp['token']);
+        SharedPreferences kubika = await SharedPreferences.getInstance();
+        kubika.setString('token', rsp['token']);
+        kubika.setBool('loggedIn', true);
```

Comment:
this logic is the same as the above one on L174 difference is that you're creating again an instance of SharedPreferences and we already have it because we've passed it to the constructor of this CodeVerification StatefulWidget